server/models/Notification.ts

```diff
@@ -0,0 +1,13 @@
+import { model, Schema, Document, models, Model } from 'mongoose'
+import { NotificationEmail } from 'utils/types'
+
+const NotificationEmailSchema = new Schema ({
+    emails: {
+        type: 'object',
```

Comment:
type should be `[String]` to show that it is an array of strings https://mongoosejs.com/docs/schematypes.html#arrays. Yeah, mongoose is weird.